dependencies.repos

```diff
@@ -0,0 +1,12 @@
+- git:
+    local-name: ros_industrial_cmake_boilerplate
+    uri: https://github.com/ros-industrial/ros_industrial_cmake_boilerplate.git
+    version: 0.5.4
```

Comment:
This is a much lower version than the head of ROS-I's `tesseract` library is using ([0.7.3 at time of posting](https://github.com/tesseract-robotics/tesseract/blob/master/dependencies.repos#L4)). Is this dependency's API stable enough that building both of these packages will be possible in the same workspace?